frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { Pagination } from '@/_components';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+const UsersTable = ({
+  isLoading,
+  users,
+  archivingUser,
+  unarchivingUser,
+  meta,
+  generateInvitationURL,
+  invitationLinkCopyHandler,
+  unarchiveOrgUser,
+  archiveOrgUser,
+  pageChanged,
+  darkMode,
+}) => {
+  const tableRef = React.createRef(null);
+
+  function calculateOffset() {
+    const elementHeight = tableRef.current.getBoundingClientRect().top;
+    return window.innerHeight - elementHeight;
+  }
+
+  const setWidth = () =>
+    (document.querySelector('.users-pagination').style.width = `${tableRef.current.offsetWidth}px`);
+
+  React.useEffect(() => {
+    window.addEventListener('resize', setWidth);
+    return () => window.removeEventListener('resize', setWidth);
+  });
```

Comment:
I think we need to add an empty array in the dependancy list or this will lead to adding a new event listener on every render